File: 3621-minimum-operations-to-make-array-values-equal-to-k/minimum-operations-to-make-array-values-equal-to-k.java
class Solution {
    public int minOperations(int[] nums, int k) {
        Map<Integer,Integer>map=new HashMap<>();
        for(int n:nums){
            System.out.println(n);
            if(n<k) return -1;
            else if(n==k) continue;
            else{
                map.put(n,map.getOrDefault(n,0)+1);
            }
        }
        return map.size();
    }
}

File: 3621-minimum-operations-to-make-array-values-equal-to-k/minimum-operations-to-make-array-values-equal-to-k.py
class Solution(object):
    def minOperations(self, nums, k):
        """
        :type nums: List[int]
        :type k: int
        :rtype: int
        """
        mp={}
        for i in nums:
            if i<k: return -1
            elif i==k: continue
            else : mp[i]= mp.get(i,0)+1
        return len(mp)